build.gradle

```
@@ -189,9 +189,16 @@ dependencies {
     implementation "io.swagger.parser.v3:swagger-parser-core:${swaggerVersion}"
     implementation "io.swagger.parser.v3:swagger-parser:${swaggerVersion}"
     implementation "io.swagger.parser.v3:swagger-parser-v3:${swaggerVersion}"
-    implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
-    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}"
-    implementation "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
+    // Declare and force Jackson dependencies for tests
+    testImplementation("com.fasterxml.jackson.core:jackson-databind") {
+        version { strictly("2.18.0") }
```

Similar for the other 2.
```suggestion
        version { strictly("${jacksonVersion}") }
```

---

Note in `resolutionStrategy` we already have
```groovy
force("com.fasterxml.jackson.core:jackson-core:2.18.0")
```
why not just add these there?   Claude tellls me we can do this:
```groovy
configurations.all {
    resolutionStrategy {
        force("com.google.guava:guava:33.3.1-jre") // CVE for 31.1, keep to force transitive dependencies
        force("org.apache.httpcomponents.core5:httpcore5:5.3") // Dependency Jar Hell
        
        eachDependency { DependencyResolveDetails details ->
            if (details.requested.group == 'com.fasterxml.jackson.core') {
                details.useVersion '2.18.0'
            }
        }
    }
}
```

---

I tried adding in the configuration block. In fact that's what I suggested @junweid62 but that fails with the below
```
 java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/Module
```
because these dependencies are needed during the implementation.

---

Can you also add the variable to the version forcing for the upstream dependency we don't use here (but do in 2.x):
```
force("com.fasterxml.jackson.core:jackson-core:2.18.0"
```